olclient/schemata/import.schema.json

```
@@ -139,31 +139,100 @@
       "properties": {
         "name": {
           "type": "string",
-	  "examples": [
-	    "Hubbard, Freeman H.",
-	    "Joan Miró"
-	  ]
-	},
-	"personal_name": {
-	  "type": "string",
-	  "description": "Can be identical to 'name'. TODO: provide information on the intended difference."
-	},
-	"birth_date": {
-	  "type": "string",
-	  "examples": []
-	},
-	"death_date": {
-	  "type": "string",
-	  "examples": []
-	},
-	"entity_type": {
-	  "type": "string",
-	  "enum": ["person", "org", "event"]
-	},
-	"title": {
-	  "type": "string",
-	  "examples": ["duc d'Otrante"]
-	}
+          "examples": [
+            "Hubbard, Freeman H.",
+            "Joan Miró"
+          ]
+        },
+        "personal_name": {
+          "type": "string",
+          "description": "Can be identical to 'name'. TODO: provide information on the intended difference."
+        },
+        "birth_date": {
+          "type": "string",
+          "examples": []
+        },
+        "death_date": {
+          "type": "string",
+          "examples": []
+        },
+        "entity_type": {
+          "type": "string",
+          "enum": ["person", "org", "event"]
+        },
+        "title": {
+          "type": "string",
+          "examples": ["duc d'Otrante"]
+        },
+        "identifiers": {
+          "type": "object",
+          "properties": {
```

This should probably allow for `additionalProperties` since adding more identifiers happens more rapidly than updating this schema. :) (E.g., since this PR, there are now at least also `project_runeberg` and `gnb` identifiers for Authors.)

```suggestion
          "additionalProperties": true,
          "properties": {
```